src/ensembl/src/content/app/browser/browserState.ts

```diff
@@ -23,6 +23,8 @@ export type BrowserNavStates = [
 
 export type ChrLocation = [string, number, number];
 
+export type BrowserChrLocation = { [ensObjectId: string]: ChrLocation };
```

Comment:
the word "browser" in `BrowserChrLocation` doesn't really help much, does it?
We used to have a single chromosome location; now we have multiple chromosome locations keyed by genomeId; does it mean that the type is `ChrLocations`? Which probably means that the `chrLocation` field of the state becomes `chrLocations`?

Also, I don't know whether it's better to have a state that looks like this:

```
{
  chrLocations: {
    id1: [1, 234, 345]
    id2: [1, 334, 445]
  },
  defaultChrLocations: {
    id1: [1, 234, 345]
    id2: [1, 334, 445]
  }
}
```

or:

```
{
 id1: {
     chrLocation: [1, 234, 345],
     defaultChrLocation: [1, 234, 345]
  },
 id2: {
     chrLocation: [1, 334, 445],
     defaultChrLocation: [1, 334, 445]
  },
}
```

It's all so confusing. Can you offer any suggestions, @ens-ds23 ?

Reply:
`ChrLocations` and `ChrLocation`? Wouldn't it be hard to distinguish?

Reply:
Don't know. But the name `BrowserChrLocation` doesn't indicate in any way how this is different from a `ChrLocation` (which is also used by genome browser, so, in a sense, it is also a `BrowserChrLocation`).